app/scripts/directives/bindService.js

```diff
@@ -69,17 +73,31 @@
     };
 
     var showBind = function() {
-      ctrl.nextTitle = 'Bind';
+      ctrl.nextTitle = bindParametersStep.hidden ? 'Bind' : 'Next >';
+      if (!selectionValidityWatcher) {
+        selectionValidityWatcher = $scope.$watch("ctrl.selectionForm.$valid", function(isValid) {
```

Comment:
I'm tempted to suggest we start using a pattern of `var scopeWatches = []` and a loop in `$onDestroy` to deregister, just to make all of our components consistent.

Reply:
Technically the scope watches clean themselves up on destroy and don't need to be explicitly deregistered. I wonder if it's worth tracking them except in the cases where we want to disable / enable them on certain steps.

Reply:
Yeah prob not so much.  I think I'm pointlessly tracking one in the drawer, should prob nix it.